pkg/sdk/data_types.go

```diff
@@ -89,6 +89,11 @@ func ToDataType(s string) (DataType, error) {
 		return DataTypeTime, nil
 	}
 
+	vectorSynonyms := []string{"VECTOR"}
```

Comment:
there is a data_types_test.go and as I see it was not added there.

Reply:
True, I missed that somehow. Adding test cases.